.github/owners.yaml

```diff
@@ -1,35 +0,0 @@
-"internal/connector/**/*":
```

Comment:
This is removed since the repo is public, we can use [CODEOWNERS](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/6fe2c376e1ade63f6834fb59a98dff3f6770b6a2/.github/CODEOWNERS)